src/error.ts

```diff
@@ -271,9 +271,9 @@ export class DBOSUnexpectedStepError extends DBOSError {
 }
 
 const TargetWorkFlowCancelled = 27;
-export class DBOSTargetWorkflowCancelledError extends DBOSError {
+export class DBOSAwaitedWorkflowCancelledError extends DBOSError {
   constructor(readonly workflowID: string) {
-    super(`Workflow ${workflowID} has been cancelled`, TargetWorkFlowCancelled);
+    super(`Workflow ${workflowID} awaited a workflow which was cancelled`, TargetWorkFlowCancelled);
```

Comment:
Why change this?  It should be like the Python one--awaited workflow "ID" was cancelled.